styleSheet.css

```diff
@@ -29,22 +29,81 @@ ul {
   float: left;
 }
 
-.navBar__list>a{
+.navBar__list > a{
   color: white;
   display: block;
   padding: 14px 16px;
   text-align: center;
   text-decoration: none;
 }
 
-.navBar__list>a:hover{
+.navBar__list > a:hover{
   background-color: #111;
 }
 
-.navBar__list--active>a{
+.navBar__list--active > a{
   color: #4CAF50;
 }
+.navBar__list__closeBtn{
```

Comment:
Consistency in using `.navBar__list__closeBtn{` or `.navBar__list__closeBtn {`

Reply:
It took me a while to see the space

Reply:
little details 👌